product_docs/docs/epas/16/reference/oracle_compatibility_reference/epas_compat_cat_views/pg_views/pg_synonym.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: "PG_SYNONYM"
+---
+
+The `PG_SYNONYM` view defines system "synonym" relations (pg_synonym) along with the relation's initial contents, if any. PostgreSQL does not provide a system catalog for `PG_SYNONYM`.
+
+| Name         | Type | Description                                             |
+| ------------ | ---- | ------------------------------------------------------- |
+| cmax         |      |                                                         |
+| cmin         |      |                                                         |
+| synlink      | text | Name of the database link, or an empty string           |                 
+| synname      | name | Name of the synonym                                     |
+| synnamespace | oid  | Namespace where the synonym resides                     |
+| synobjname   | text | Name of the base object referenced by the synonym       |
+| synobjschema | text | Name of the schema where the base object resides        |
+| synowner     | oid  | Object identifier of the owner (creator) of the synonym |
+| tableoid     |      |                                                         |              
+| xmax         |      |                                                         |   
+| xmin         |      |                                                         |   
```

Comment:
in EPAS v16, we have only these columns 

edb=# \d pg_synonym
            Table "pg_catalog.pg_synonym"
    Column    | Type | Collation | Nullable | Default 
--------------+------+-----------+----------+---------
 oid          | oid  |           | not null | 
 synname      | name |           | not null | 
 synnamespace | oid  |           | not null | 
 synowner     | oid  |           | not null | 
 synobjschema | text | C         |          | 
 synobjname   | text | C         | not null | 
 synlink      | text | C         |          | :